guide/index.rst

```diff
@@ -23,34 +23,57 @@ Make sure to read this `Documentation style guide`_ as well as these
 How to contribute ?
 ===================
 
-Choose a topic that is not yet covered and write it up !
+* If you spot typos, unclear or clumsy wording in the lecture notes, please
+  help to improve them. Simple text editing can be done by `editing files
+  in your GitHub fork
+  <https://help.github.com/articles/editing-files-in-your-repository/>`_ of
+  the lecture notes.
+
+* Choose a topic that is not yet covered and write it up !
```

Comment:
Maybe here you could mention that the right way to do this is first to create an issue on github, so that there is a discussion about the scope of the corresponding chapter.